### DISCUSSION TITLE
Possibility to use DIA-NN as DDA data search engine?

### DISCUSSION THREAD

**Discussion by cctsou** [vdemichev/DiaNN#89] [Ideas]
Hi,

Any possibility or perhaps future plan that we could use DIA-NN package even for DDA data as a DDA MS/MS search engine?
I felt that DIA-NN has almost everything needed so that we could take advantage DIA-NN and predicted spectra to further improve conventional DDA search engine performance.

**Comment by vdemichev** [comment]
Hi, I recommend using FragPipe for all DDA searches. It does produces spectral libraries compatible with DIA-NN. Specifically DIA-NN will not be developed towards native DDA support in the near future - unlike DIA, DDA does not benefit so much for the use of predicted spectra & other sophisticated algorithms.
